modules/cnv-updating-access-mode-to-enable-live-migration.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * cnv/live_migration/cnv-live-migration.adoc
+
+[id="cnv-updating-access-mode-to-enable-live-migration_{context}"]
+= Updating access mode to enable LiveMigration
+
+For LiveMigration to function properly, your storage class must use the
+RWX access mode.
+
+Storage class defaults in {CNVProductName} are configured to provide
```

Comment:
This isn't entirely accurate. The default storage class uses a RWO accessMode by default, but subsequent storage classes are added to the ConfigMap by the user, in which case they specify the accessMode then.

Reply:
_Changed to the following, dropping the paragraph beginning with "Storage class defaults...."_

Default storage classes use a ReadWriteOnce (RWO) access mode by default. 
When you add additional storage classes to the `ConfigMap`, specify the
appropriate access mode. 

For LiveMigration to function properly, your storage class must use the
ReadWriteMany (RWX) access mode.